auth0_flutter_platform_interface/CHANGELOG.md

```
@@ -1,5 +1,11 @@
 # Change Log
 
+## [afpi-v2.3.0](https://github.com/auth0/auth0-flutter/tree/afpi-v2.3.0) (2026-06-15)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)